docs/troubleshooting.rst

```diff
@@ -193,3 +193,16 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+fatal: the remote end hung up unexpectedly / fatal: early EOF / fatal: index-pack failed
+----------------------------------------------------------------------------------------
+
+This issue can occur due to problems with your network while cloning edx-platform which is a fairly large repository.
+
+You should first of all try to rerun the command you were trying to run to see if it works.
```

Comment:
This is good advice, but it does not inspire confidence. How many times should the user try to re-run the same command? Why should they do that?

Reply:
I've added that the command should be run once again and why it should be run again.